.github/workflows/ci.yml

```diff
@@ -100,6 +100,48 @@ jobs:
           KERNELDIR=/lib/modules/$(ls /lib/modules)/build make -j4
           make run-unit-tests
 
+  build-shared-libs-linux-amd64:
+    name: build-shared-libs-linux-amd64 🧐
+    runs-on: ubuntu-latest
+    steps:
+      - name: Install deps ⛓️
+        run: |
+          sudo apt update
+          sudo apt install -y --no-install-recommends ca-certificates cmake build-essential git clang llvm pkg-config autoconf automake libtool libelf-dev wget libb64-dev libc-ares-dev libcurl4-openssl-dev libssl-dev libre2-dev libtbb-dev libjq-dev libjsoncpp-dev libgrpc++-dev protobuf-compiler-grpc libgtest-dev libprotobuf-dev liblua5.1-dev linux-headers-$(uname -r)
+
+      - name: Checkout Libs ⤵️
+        uses: actions/checkout@v3
+        with:
+          fetch-depth: 0
+
+      - name: Git safe directory
+        run: |
+          git config --global --add safe.directory $GITHUB_WORKSPACE
+
+      - name: Build and test 🏗️🧪
+        run: |
+          mkdir -p build
+          cd build && cmake -DBUILD_SHARED_LIBS=True -DUSE_BUNDLED_DEPS=False -DMINIMAL_BUILD=True -DUSE_BUNDLED_VALIJSON=True -DCMAKE_INSTALL_PREFIX=/tmp/libs-test ../
+          KERNELDIR=/lib/modules/$(ls /lib/modules)/build make -j4
+          make run-unit-tests
+          
+      - name: Install
+        run: |
+          cd build
+          make install
+          
+      - name: Test sinsp-example build with pkg-config
+        run: |
+          cd userspace/libsinsp/examples
+          export PKG_CONFIG_PATH=/tmp/libs-test/lib/pkgconfig
+          g++ -o sinsp-example test.cpp util.cpp $(pkg-config --cflags --libs libscap libsinsp)
```

Comment:
Just a minor note: shouldn't libsinsp only be ok? I mean, libsinsp alone should carry also libscap deps, since it links libscap.
I am not sure though.

Reply:
The `Requires: libscap` line in libsinsp.pc should pick up libscap, and it does so here. I removed the explicit libscap.